transitions/index.js

```diff
@@ -29,70 +29,35 @@ var availableTransitions = [
   'update_sent_forms'
 ];
 
-/*
- * Supported configuration:
```

Comment:
it was really nice to get a list of supported formats! Here or somewhere else, it's useful.